csharp/tools/tracing-config.lua

```diff
@@ -115,29 +185,28 @@ function RegisterExtractorPack(id)
             end
         end
         if match then
-            local injections = { '-p:UseSharedCompilation=false', '-p:EmitCompilerGeneratedFiles=true' }
+            -- TODO: Add real temporary file name
+            local injections = { '-bl:codeql_' .. RandomString(15) .. '.binlog' }
```

Comment:
Passing `-bl` to `dotnet run` doesn't seem to do anything, but it makes the extractor fail due to the non existent binlog file.